test/python/runCythonTests.py

```diff
@@ -22,50 +22,29 @@
 import os
 from pathlib import Path
 
-cantera_root = os.path.relpath(__file__).split(os.sep)[:-1] + ['..', '..']
-os.chdir(os.sep.join(cantera_root + ['test', 'work']))
+CANTERA_ROOT = Path(__file__).parents[2]
+os.chdir(str(CANTERA_ROOT / "test" / "work"))
```

Comment:
I'm not a big fan of `chdir`, I find that it leads to confusion about where files will be placed. Hopefully we can just get rid of this file entirely someday, since we don't need all the stuff that we had for compatibility with `unittest` that are in this file.